lua/bufferline/api.lua

```diff
@@ -65,10 +66,10 @@ local function pick_buffer_wrap(fn)
 end
 
 --- Shows an error that `bufnr` was not among the `state.buffers`
---- @param bufnr integer
-local function notify_buffer_not_found(bufnr)
+--- @param buffer_number integer
+local function notify_buffer_not_found(buffer_number)
   notify(
-    'Current buffer (' .. bufnr .. ") not found in bufferline.nvim's list of buffers: " .. vim.inspect(state.buffers),
+    'Current buffer (' .. buffer_number .. ") not found in bufferline.nvim's list of buffers: " .. vim.inspect(state.buffers),
```

Comment:
I had to rename a bunch of these, because `vim.fn.bufnr` is now imported at the top